docs/docs/user-guide/using-own-identity.mdx

```diff
@@ -5,11 +5,11 @@ sidebar_position: 10
 
 # Use your own Microsoft Entra identity
 
-When you use the CLI for Microsoft 365 to connect to your tenant for the first time, you are presented with a `Permissions requested` prompt from Azure, by accepting this prompt you are consenting to using the `PnP Microsoft 365 Management Shell` Microsoft Entra application with your tenant as well as the permissions that it requires.
+When you use the CLI for Microsoft 365 to connect to your tenant for the first time, you need to specify at least the `appId` and `tenant` options during login.
```

Comment:
This is true, unless you used `m365 setup`

Reply:
The section below already mentions that you can use the `m365 setup` command to simplify this. Is that enough, or should it be mentioned here as well?

Reply:
Since we're reading top to bottom, I suggest we mention setup first.